docs/faq.md

```diff
@@ -0,0 +1,95 @@
+# FAQ
+
+## Reusing already running Cluster
+
+Running a test with already provisioned cluster, you can get the test_id in the AWS console or from jenkins logs:
+
+```
+# add the following to your config yaml
+reuse_cluster: 7c86f6de-f87d-45a8-9e7f-61fe7b7dbe84
+
+# or with using the new configuration, before the run test command
+export SCT_REUSE_CLUSTER=7c86f6de-f87d-45a8-9e7f-61fe7b7dbe84
+```
+
+## using SCT_UPDATE_DB_PACKAGES to override AMI with newer binaries
```

Comment:
switch those question to something can be a link, since we keep send those out to people (before in the rst it wasn't a link)